recipes/graphlan/meta.yaml

```diff
@@ -3,7 +3,7 @@ about:
     summary: "GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation."
 package:
   name: graphlan
-  version: '1.0.0'
+  version: '0.9.7'
```

Comment:
Why is that? You are still downloading the `hg_tag: 1.0`.
The problem here is that conda will take the latest version by default. So `conda install graphlan` will install version 1.0 I guess.